Content.Server/Disease/DiseaseSystem.cs

```diff
@@ -97,6 +97,13 @@ public override void Update(float frameTime)
                 {
                     var disease = carrierComp.Diseases[i];
 
+                    //if the disease is on the silent disease list, don't do effects
+                    var doEffects = true;
+                    if (carrierComp.SilentDiseases != null)
+                        foreach (var id in carrierComp.SilentDiseases)
+                            if (id == disease.ID)
+                                doEffects = false;
```

Comment:
The concept of SilentDiseases feels pretty dirty to me, at least make this a hashset. Maybe also call it "CarrierDiseases" or something.